make.sh

```diff
@@ -59,14 +59,14 @@ if [[ -f $BUILD_COMMIT_ID_FILE ]]; then
 fi
 
 echo_stage "Building LightGBM CI docker image replica..."
-bash make_docker_image.sh
+bash docker/make_docker_image.sh
```

Comment:
Same as above ☝️

Reply:
Got your point :), I left `docker/make_docker_image.sh` outside the `docker/lightgbm-ci-build-env` so it's clear that that folder is what's used for the full docker environment setup + scripts.